modules/building/pages/apply-task-migrations.adoc

```diff
@@ -0,0 +1,113 @@
+= Applying task migrations
+
+{ProductName} allows task owners to write migrations to bring pipeline changes
+to users. Generally, users do not need to care about how the migrations are
+defined and how {ProductName} applies migrations to their build pipelines.
+MintMaker is configured well and the updates are sent regularly along with
+Renovate update pull request together. However, in case users requires to apply
+migrations by themselves, this document illustrates how to do that.
+
+== Apply migration with pipeline-migraiton-tool
+
+. Install pipeline-migration-tool:
+
++
+Choose a release from
+https://github.com/konflux-ci/pipeline-migration-tool/releases[releases] page
+and install a specific version.
+
++
+[source,bash]
+----
+pipx install https://github.com/konflux-ci/pipeline-migration-tool/archive/refs/tags/v0.4.2.zip
+----
+
++
+then, there are two executables available in the `$PATH`,
+`pipeline-migration-tool` and `pmt`. Each of them is usable. In this document,
+`pmt` will be used.
```

Comment:
Slight rewording to make it clear the `pipeline-migration-tool` and `pmt` are the same thing and to use active voice rather than passive voice

```suggestion
This installs two executables in `$PATH` - `pipeline-migration-tool` and its alias, `pmt`.
In this document, we will use `pmt`.
```